### ISSUE TITLE
pypi release missing geopandas integration

### ISSUE THREAD

**Issue by cmollet** [remix/partridge#57]
* partridge version: 1.0.0
* Python version: anaconda3-5.3.1
* Operating System: macOS 10.14.4

### Description

Greetings from the [American Planning Association](https://github.com/apa-dev)! We're in San Francisco for our [National Planning Conference](https://planning.org/conference/). One component of this is a [DataJam](https://www.eventbrite.com/e/get-sf-transit-moving-data-jam-npc19-tickets-59251933127) meant to get planners and civic tech people working together to try to solve some San Francisco transit problems.

I'm putting together some JupyterHub-hosted notebooks bootstrapped with data so that people can get working with data right away and not fritter away valuable time messing with getting environments and dependencies set up, ETL-ing and cleaning data, etc. `partridge` looks like an awesome resource for working with GTFS data efficiently without having to deploy a database. We'd love to get DataJam attendees using it as one possible method for producing awesome transit analyses and visualizations. 

The `geopandas` integration looks like it hasn't made it to [pypi](https://pypi.org/project/partridge/) yet. There's no `full` in `setup.py` extras; `geo.py` is missing and the `geopandas` dependency was not installed. 

Installing using `pip`'s VCS method from this repo works just fine, e.g. `pip install -e git+https://github.com/remix/partridge/#egg=partridge`, and I've used that to get it working in our hosted JupyterHub on [Azure Notebooks](https://notebooks.azure.com) so DataJam attendees can use it without problems during the event. Just thought you'd want the heads up!

### What I Did

```
pip install "partridge[full]"
Collecting partridge[full]
  Using cached https://files.pythonhosted.org/packages/19/a6/3a26b6ffc3a317248a279f9c0057ae92e9f3432af50af8d233c217d80de3/partridge-1.0.0-py2.py3-none-any.whl
  partridge 1.0.0 does not provide the extra 'full'

[extraneous output ommitted]
```

**Comment by invisiblefunnel** [comment]
Hi @cmollet. I need to release a new version. Will take care of that today or tomorrow.

**Comment by invisiblefunnel** [comment]
I've released `v1.1.0`. Thanks!

**Comment by cmollet** [comment]
Perfect, thanks!
